src/main/java/com/icthh/xm/ms/entity/service/swagger/model/SwaggerModel.java

```diff
@@ -13,8 +14,9 @@
 
 import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
 
+@EqualsAndHashCode(callSuper = true)
 @Data
-public class SwaggerModel {
+public class SwaggerModel extends com.icthh.xm.commons.swagger.model.SwaggerModel {
```

Comment:
what inside SwaggerModel in commons?
can we just move this model to commons? (but maybe in next PR)

Reply:
it's empty now, extending here exists for entity compatibility. This code will be removed, after swagger generation related code moved to commons